### PR TITLE
add balance banner stories

### PR DESCRIPTION
**Pull request by wolverineks**

### CODE REVIEW

**Comment by wolverineks** [comment]
check the error story

File: src/hooks/index.ts [hunk 1]
```diff
@@ -648,9 +648,8 @@ export const useExchangeRate = ({
   const query = useQuery<number, Error>({
     suspense: true,
     staleTime: 60000,
-    retry: 3,
     retryDelay: 1000,
-    queryKey: [to],
+    queryKey: [wallet.id, 'exchangeRate', {to}],
```
Comment (stackchain) [inline src/hooks/index.ts:652]: I think we should leverage the query cache across the wallets.

Comment (stackchain) [reply src/hooks/index.ts:652]: our backend sometimes is very unstable due to the load, and often we receive some errors, perhaps a few retries could save the user from a bad experience on the first load

Comment (wolverineks) [reply src/hooks/index.ts:652]: what about ergo wallets?

Comment (wolverineks) [reply src/hooks/index.ts:652]: cache across network?

Comment (wolverineks) [reply src/hooks/index.ts:652]: either way, each key should be loggable / readable

Comment (stackchain) [reply src/hooks/index.ts:652]: the symbol will be different for the default asset ERG, the network won't affect the price

Comment (stackchain) [reply src/hooks/index.ts:652]: > either way, each key should be loggable / readable

easily convinced 👍

**Review by stackchain — APPROVED** [review]

**Pull request merged**